### DISCUSSION TITLE
Building wheel for multidict (PEP 517) error from discord.py pip

### DISCUSSION THREAD

**Discussion by SundaePixel** [Rapptz/discord.py#6036] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74124581/98974598-b6023300-24da-11eb-811d-3a686c4245cc.png)

I got no idea why this is happening. Can some help help me out?

**Comment by ShineyDev** [comment, marked as answer]
This is a duplicate of #6011.

**Reply by SundaePixel** [reply]
> This is a duplicate of #6011.

Thank you!
